tests/storage/fsp/conftest.py

```diff
@@ -1,24 +1,55 @@
 import pytest
 
+FSP_REPO_FILE = '/etc/yum.repos.d/xcp-ng-runx.repo'
+
+FSP_REPO_TEXT = """\
+[xcp-ng-runx]
+name=XCP-ng runx Repository
+baseurl=http://mirrors.xcp-ng.org/8/8.2/runx/x86_64/ http://updates.xcp-ng.org/8/8.2/runx/x86_64/
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-xcpng
+"""
+
+FSP_PACKAGES = ['xcp-ng-xapi-storage']
+
 DIRECTORIES_PATH = 'directories'
 
+def install_fsp(host):
+    assert not host.file_exists('/usr/bin/runx'), \
+        "runx should not be installed on the host before all tests"
+    host.create_file(FSP_REPO_FILE, FSP_REPO_TEXT)
```

Comment:
In addition to `create_file`, a Host method to add an optional xcp-ng yum repository (runx, linstor, ...) would be nice.

Example signature :

```
    def add_xcpng_repo(name):
        # creates the repo file for repo `name` from XCP-ng mirror
```